### DISCUSSION TITLE
Time control bar enhanced with sidereal time?

### DISCUSSION THREAD

**Discussion by diegorodriguezperegrin** [Stellarium/stellarium#2683] [Ideas]
The current time controls located in the bottom tool bar work very well, but lack sidereal time. I guess it would be pretty easy to add, so users wouldn't need to use keys combinations that are hard to compose (pressing simultaneously several keys), remember or know for different language keyboard layouts. 

The possibilities between solar and sidereal and steps already present as key combinations may be easily ported to the graphical interface just adding a third state to PLAY button: PAUSE, SOLAR, SIDEREAL. By pressing Sidereal time, time stamp above would show LST and all time steps already present would become sidereal rates (day, week, month, year).  A similar option may be added to the floating time control bar.

By using sidereal rates, many sky movements may become more apparent.

Many thanks for reading this.

**Comment by gzotti** [comment]
LST=Local Solar Time, not sidereal.

You can configure hotkeys to your liking. 

One fundamental problem of sidereal time is that it is not unambiguous. If you want to work with date&sidereal time, setting ST which corresponds to 0:01 is the same as that corresponding to 23:57. So this interface will not work.

A most interesting function has been introduced recently and described in the User Guide, ch.4.8.1.

**Reply by diegorodriguezperegrin** [reply]
Not sure what you mean by "LST=Local Solar Time, not sidereal".

Anyway, I think I am starting to understand the reasoning behind leaving all options (sidereal, draconic, anomalistic, etc) measures as part of the available shortcuts. Although the "ambiguity" of sidereal time may be a matter of warning the user, and conversion between mean sidereal and mean solar time may be derived from the published square polynomial or from the more modern ERA simple linear polynomia, it's probably more efficient to move all those specific time modes to the keyboard shortcuts instead.

Yes, the keyboard shorcuts are a new marvelous tool quickly avalable in the Help window. If going through Configuration→**Enable Keyboard navigation**→Edit keyboard shortcuts... maybe using Configuration→**Enable keyboard shortcuts**→Edit keyboard shortcuts... may look more consistent.

Altogether, Stellarium is becoming more and more a weapon of choice to all astronomy lovers and pros from any background. Heartfelt I have learnt much more from it that from many brainy books. Many thanks for this incredible enterprise and congratulations!

**Reply by gzotti** [reply]
You suggested to operate in Sidereal Time and that the time in the menu bar should become LST. I assumed a confusion of our timezone setting LMST or LTST. The S in these mean "Solar". 

We provide Sidereal time in the regular infostring for any selected object.

Keyboard navigation means moving with the cursor keys. Keyboard shortcuts work even if this is disabled. But yes, the config button for KB shortcuts should not be blocked when KB Nav is disabled. @alex-w ?

Thank you for the comment. Although "tool of choice" would sound nicer in this world. Weapons and astronomy don't fit together IMHO.

**Reply by diegorodriguezperegrin** [reply]
"Thank you for the comment. Although "tool of choice" would sound nicer in this world. Weapons and astronomy don't fit together IMHO." GRANTED! ;)
